pages/mypage/collection.tsx

```diff
@@ -0,0 +1,177 @@
+import Footer from '@components/common/Footer';
+import Header from '@components/common/Header';
+import MySongItem from '@components/mypage/mysong/MySongItem';
+import MyVocab from '@components/mypage/mysong/MyVocab';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { mockClient } from 'lib/api';
+import { useRouter } from 'next/router';
+import React, { ReactElement, useEffect, useState } from 'react';
+import useSWR from 'swr';
+interface StyledProps {
+  mySongs: boolean | undefined;
+  myVocab: boolean | undefined;
+}
+
+export interface mySongItem {
+  id: number;
+  title: string;
+  artist: string;
+  albumImageUrl: string;
+}
+
+function Collection(): ReactElement {
+  const router = useRouter();
+  const pid = router.query;
+  const [mySongs, setmySongs] = useState<boolean | undefined>();
+  const [myVocab, setmyVocab] = useState<boolean | undefined>();
```

Comment:
기본 router를 mysongs로 설정할까?
```suggestion
  const [mySongs, setmySongs] = useState<boolean>(true);
  const [myVocab, setmyVocab] = useState<boolean>(false);
  useEffect(() => {
    router.push('/mypage/collection?type=mysongs');
  }, []);
```

Reply:
아아 마이페이지 들어갈떄 쿼리 붙여서 들어가는걸로 합시다~

Reply:
굿굿~ history.push 할 때 쿼리값 디폴트로 마이송으로 넣으면 될듯!!